src/main/kotlin/pl/edu/agh/gem/config/AuthConfig.kt

```diff
@@ -0,0 +1,25 @@
+package pl.edu.agh.gem.config
+
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.security.config.annotation.web.builders.HttpSecurity
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder
+import org.springframework.security.crypto.password.PasswordEncoder
+import org.springframework.security.web.SecurityFilterChain
+
+@Configuration
+@EnableWebSecurity
+class AuthConfig {
+    @Bean
+    fun encoder(): PasswordEncoder {
+        return BCryptPasswordEncoder()
+    }
+
+    @Bean
+    fun securityFilterChain(http: HttpSecurity): SecurityFilterChain {
```

Comment:
do we still need spring security configuration if we want to permit any request ?

Reply:
yes, its a side effect of using spring security for authentication. Without this filter all request would be blocked